c/array-patterns/array13_pattern.c

```diff
@@ -29,16 +29,17 @@ void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
 extern int __VERIFIER_nondet_int() ;
 extern short __VERIFIER_nondet_short() ;
 
-short ARR_SIZE ;
+signed long long ARR_SIZE ;
 
 int main()
 {
-	ARR_SIZE = __VERIFIER_nondet_short() ;
+	ARR_SIZE = (signed long long)__VERIFIER_nondet_short() ;
 	__VERIFIER_assume(ARR_SIZE > 0) ;
 
 	int array[ARR_SIZE][ARR_SIZE] ;
 	
-	int row = 0, column = 0, num = -1, sum = 0 ;
+	int row = 0, column = 0, num = -1;
+       	signed long long sum = 0 ;
```

Comment:
I don't think `sum` needs to be larger than an `int`?

Reply:
Shouldn't be. I wanted to make sure the data types of all these variables to be consistent accross all benchmarks in this folder. That's why, opted to go for type promotion to signed long long.